ros_buildfarm/argument.py

```diff
@@ -474,6 +474,14 @@ def add_argument_testing(parser):
              'them.')
 
 
+def add_argument_return_zero(parser):
```

Comment:
I gather that earlier iterations of this PR had the new audit scripts run on the buildfarm and used as an additional guard on sync jobs. But since that is no longer the case is this argument still needed?

Reply:
This argument was added for being able to run on Travis and differentiate between a completed report vs a crash.

Reply:
Instead of adding an argument for this I would argue the script should be invoked in a way which ignores the return code (e.g. `... || true`).

Reply:
I initially thought of using `|| true` when adding it to travis. However, I realized that would also mask any crashes and make it not useful to run the test on travis at all since it would never fail. Thus I took the extra time to implement the additional argument which will return zero if the program completes successfully but had a failing report which is distinct from what the unit test cares about which is crashing and failing to complete which will still generate a non-zero return code.